src/quo2/components/messages/system_message.cljs

```diff
@@ -83,51 +75,48 @@
       [text/text
        {:size  :paragraph-2
         :style {:color (get-color :text)}}
-       (or (get labels label)
-           label
-           (:message-deleted labels))])
-    [sm-timestamp timestamp-str]]])
+       (or label (i18n/label :message-deleted))])]])
 
-(defmethod system-message-content :added
-  [{:keys [state mentions timestamp-str labels]}]
+(defn system-message-contact
+  [{:keys [display-name photo-path]} label]
   [rn/view
    {:align-items    :center
     :flex-direction :row}
    [sm-icon
-    {:icon    :main-icons/add-user
+    {:icon    :i/add-user
      :color   :primary
-     :opacity (if (= state :landed) 0 5)}]
-   [sm-user-avatar (:image (first mentions))]
+     :opacity 5}]
+   [sm-user-avatar display-name photo-path]
    [text/text
     {:weight :semi-bold
      :size   :paragraph-2}
-    (:name (first mentions))]
+    display-name]
    [text/text
     {:size  :paragraph-2
      :style {:color        (get-color :text)
              :margin-left  3
              :margin-right 3}}
-    (:added labels)]
-   [sm-user-avatar (:image (second mentions))]
-   [text/text
-    {:weight :semi-bold
-     :size   :paragraph-2}
-    (:name (second mentions))]
-   [sm-timestamp timestamp-str]])
+    label]])
+
+(defmethod system-message-content :added
+  [data]
+  [system-message-contact data "is now a contact"])
+
+(defmethod system-message-content :removed
+  [data]
+  [system-message-contact data "removed you as a contact"])
 
 (defmethod system-message-content :pinned
-  [{:keys [state pinned-by child timestamp-str labels]}]
+  [{:keys [state pinned-by child]}]
   [rn/view
    {:flex-direction :row
     :flex           1
     :align-items    :center}
    [sm-icon
-    {:icon    :main-icons/pin
+    {:icon    :i/pin
      :color   :primary
```

Comment:
I think this use of primary color is wrong - it should be based on `customization-color`

Reply:
could you elaborate?

Reply:
in the designs the color here is based upon users color (or probably chat color too for groups etc)
<img width="347" alt="Screenshot 2023-07-26 at 15 03 32" src="https://github.com/status-im/status-mobile/assets/22799766/35b7d0c7-9fe3-4b07-b19a-80a6edda5498">

so probably this should be 
```
:color (colors/theme-color 
(colors/custom-color customization-color 50 5)
(colors/custom-color customization-color 60 5)
theme
```
where customization-color is a prop passed in

Reply:
it's handled in `icon-avatar` component